src/test/polymer-project_test.ts

```diff
@@ -147,45 +147,166 @@ suite('PolymerProject', () => {
 
   });
 
-  test('splits and rejoins scripts', (done) => {
+  suite('splitter/rejoiner', () => {
     const splitFiles = new Map();
     const joinedFiles = new Map();
-    defaultProject.sources()
+    const expectedSplitFiles: String[] = [];
+    const expectedJoinedFiles: String[] = [];
+
+    suiteSetup((done) => {
+      defaultProject.sources()
         .pipe(defaultProject.splitHtml())
         .on('data', (f: File) => splitFiles.set(unroot(f.path), f))
         .pipe(defaultProject.rejoinHtml())
         .on('data', (f: File) => joinedFiles.set(unroot(f.path), f))
         .on('end', () => {
-          const expectedSplitFiles = [
+          /* no spread operator in Node 4, so use apply... */
+          Array.prototype.push.apply(expectedSplitFiles, [
```

Comment:
`expectedSplitFiles` is always empty at this point, why use push here instead of just setting the array?